container_engine/django_tutorial/Dockerfile

```diff
@@ -21,11 +21,11 @@ FROM gcr.io/google_appengine/python
 
 # Create a virtualenv for the application dependencies.
 # # If you want to use Python 3, add the -p python3.4 flag.
```

Comment:
We should change this comment, since python36 is preselected.

Reply:
Thanks, I changed it.